packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,24 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+
+/**
+ * Default config for [InputSpotlightDecorator]{@link moonstone/Input.InputSpotlightDecorator}
+ *
+ * @memberof moonstone/Input/InputSpotlightDecorator.InputSpotlightDecorator
```

Comment:
There were some doc parsing errors due to the pathing change above not matched everywhere in the module. While I think what you had `moonstone/Input.InputSpotlightDecorator` is probably "more right", I erred on the side of fewer changes and aligned the new docs with the existing naming.

Reply:
I would prefer to leave it the correct way.  It errors because the decorator is private, so this should be, too.  If we do that, the warnings should go away.

Reply:
Ugh, I see.  Well, just leave it as-is, then.